ams_flutter/lib/features/onboarding/presentation/pages/profile_student.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:flutter/material.dart';
+import 'package:ams_flutter/core/constants/app_text_styles.dart';
+import 'package:ams_flutter/core/constants/app_images.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/bottom_navbar.dart';
+
+class ProfilePage1 extends StatefulWidget {
+  @override
+  State<ProfilePage1> createState() => _ProfilePage1State();
+}
+
+class _ProfilePage1State extends State<ProfilePage1> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              _buildHeader(context),
+              SizedBox(height: 16.0),
+              _buildProfileDetails(context),
+              SizedBox(height: 16.0),
+              _buildAttendance(context),
+            ],
+          ),
+        ),
+      ),
+      bottomNavigationBar: BottomNavBar(),
+    );
+  }
+
+  Widget _buildHeader(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Image.asset(
+          AppImages.arrow_image,
+          width: MediaQuery.of(context).size.width * 0.06,
+          height: MediaQuery.of(context).size.width * 0.06,
+        ),
+        Text(
+          'Your Profile',
+          style: TextStyles.profilePageHeader,
+        ),
+        Image.asset(
+          AppImages.bell_image,
+          width: MediaQuery.of(context).size.width * 0.06,
+          height: MediaQuery.of(context).size.width * 0.06,
+        ),
+      ],
+    );
+  }
+
+  Widget _buildProfileDetails(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.all(16.0),
+      decoration: BoxDecoration(
+        border: Border.all(color: Color(0xff9b9b9b)),
+        color: Color(0xfff8faf9),
+        borderRadius: BorderRadius.circular(10.0),
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Center(
+            child: Container(
+              width: MediaQuery.of(context).size.width * 0.25,
+              height: MediaQuery.of(context).size.width * 0.25,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(50),
+                border: Border.all(color: Color(0xffb4b4b4)),
+                image: DecorationImage(
+                  fit: BoxFit.cover,
+                  image: AssetImage(
+                    AppImages.dp_image,
+                  ),
+                ),
+              ),
+            ),
+          ),
+          SizedBox(height: 16.0),
+          _buildProfileRow('Name', 'Steve'),
```

Comment:
Use a `List<Map<T,T>>` to store the dummy values, and run `map` method on it to return the corresponding widget. Reduces code redundancy, and makes it look cleaner

Reply:
done